style.css

```diff
@@ -1,336 +1,764 @@
+    margin-bottom: 50px;
+  }
 
-.close {
-  margin-left: 25px;
-  margin-top: 50px;
-  background: #fff;
-  border: none;
-}
+  .about-li {
+    display: flex;
+    flex-direction: column;
+  }
 
-.closure {
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  margin-right: 25px;
-}
+  .ab-head {
+    color: #344563;
+    font-size: 40px;
+    margin-left: 80px;
+    margin-left: 200px;
+    margin-top: 10px;
+  }
 
-.ab-form {
-  margin-left: 25px;
-}
+  .side2 {
+    margin-top: 150px;
+  }
 
-#abt-contact {
-  display: flex;
-  flex-direction: column;
-}
+  .ab-text {
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    font-weight: 400;
+    color: #42526e;
+  }
 
-input[type="text"] {
-  height: 3rem;
-  width: 20rem;
-  margin-bottom: 2rem;
-  border-radius: 6px 6px 6px 6px;
-  border: #cfd0d1 solid 1px;
-  padding-left: 1rem;
-  font-size: 1rem;
-}
+  .resume {
+    width: 200px;
+    margin-left: 200px;
+  }
 
-input[type="email"] {
-  height: 3rem;
-  width: 20rem;
-  margin-bottom: 2rem;
-  border-radius: 6px 6px 6px 6px;
-  border: #cfd0d1 solid 1px;
-  padding-left: 1rem;
-  font-size: 1rem;
-}
+  .ab-lang {
+    background: #ebf0ee;
+    height: 352px;
+    margin-top: 50px;
+    margin-left: 25px;
+  }
 
-input[type="submit"] {
-  padding-bottom: 1rem;
-  margin-top: 30px;
-}
+  .lang {
+    width: 300px;
+  }
 
-.footer {
-  display: flex;
-  height: 10px;
-}
+  .flex-tool {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+  }
 
-.footer-img {
-  margin-left: 70px;
-  margin-bottom: 150px;
-}
+  .ab-btn {
+    background: #fff;
+    color: #36b37e;
+    display: flex;
+    padding-left: 15px;
+    padding-right: 10px;
+    padding-top: 10px;
+  }
 
-.ab-social {
-  margin-top: 50px;
-}
+  .container {
+    margin-left: 70px;
+  }
 
-.ab-soc {
-  font-size: 25px;
-  margin-right: 40px;
-  color: #42526e;
-}
+  .flex-con {
+    display: flex;
+  }
 
-.ab-mail {
-  text-align: center;
-  font-size: 20px;
-  color: #42526e;
-}
+  .test {
+    width: 270px;
+    height: 350px;
+    color: #36b37e;
+    margin: 10px;
+    border-radius: 8px 8px 8px 8px;
+    background: #ebf0ee;
+  }
 
-textarea {
-  height: 8rem;
-  width: 20rem;
-  border-radius: 6px 6px 6px 6px;
-  border: #cfd0d1 solid 1px;
-  padding-left: 1rem;
-  font-size: 1rem;
-}
+  .ab-img {
+    margin-top: 10px;
+  }
+
+  .lang-btn {
+    color: #36b37e;
+    height: 30px;
+    background: #ffff;
+    margin-left: 10px;
+    border-radius: 6px 6px 6px 6px;
+    margin-bottom: 5px;
+    border: none;
+  }
+
+  .btn-pad {
+    padding: 0;
+    list-style: none;
+  }
+
+  .lang-name {
+    padding-left: 5px;
+  }
+
+  /* ************************ section Three Above **************** */
+
+  .prologue {
+    padding-left: 100px;
+    height: 1434.3px;
+    margin-top: 20px;
+  }
+
+  .contact-me {
+    margin-top: 100px;
+  }
+
+  .contact-sec {
```

Comment:
:red_circle: You have put in great effort to achieve the design so far as requested by the previous reviewer :clap:. The content of this section is aligned on the left while it is supposed to be centered . Kindly set the` justify-content` to `center` for this element to match the Figma design

Your Design
![des](https://user-images.githubusercontent.com/26736582/200122319-62e2ddc2-f037-4d06-b4c1-839a60aeb113.png)

Figma Design
![fig](https://user-images.githubusercontent.com/26736582/200122347-17b90079-9f5c-48de-ae82-9f9603bf4245.png)